mscore/musescore.cpp

```diff
@@ -5768,7 +5768,7 @@ void MuseScore::switchLayoutMode(LayoutMode mode)
       {
       // find a measure to use as reference, if possible
       QRectF view = cv->toLogical(QRect(0.0, 0.0, width(), height()));
-      Measure* m = cs->firstMeasure();
+      Measure* m = cs->firstMeasureMM();
```

Comment:
I believe that setting _m_ to `cs->firstMeasure()` rather than `cs->firstMeasureMM()` is what was causing this particular crash.